src/components/student/TaskModalStudent.tsx

```diff
@@ -1,11 +1,13 @@
-import CloseIcon from '@mui/icons-material/Close'
```

Comment:
It seems like this file isn't being used by anything at the moment. Could you leave a comment `// ! DEPRECATED` at the top of any files that are deprecated bc of the refactoring?